sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -27,6 +27,7 @@
 import AzureCommunication
 import AzureCore
 import Foundation
+import AzureCommunicationSignaling
```

Comment:
If signaling is optional, we should surround this import and ALL code which uses it, with `#if canImport(AzureCommunicationSignaling)`